### ISSUE TITLE
[Question] Specify type for @PathParams

### ISSUE THREAD

**Issue by zoubarevm** [Romakita/ts-express-decorators#38]
Hey!

I was wondering if there are any examples converting `@PathParams` to any other primitives types (e.g. number) other than string. I can do the conversion within the controller, but I wanted to leverage the attribute in a more global way. 

E.g.

```javascript
    @Get('/:id(\\d+)')
    @Authenticated()
    public getById(@Request() request: Express.Request,
                        @PathParams('id') id: number) {
        console.log(id); // id will be of type 'string'
        return id;
    }
```

Cheers!

**Comment by Romakita** [comment]
Hi @zoubarevm ,

Actually the converter in v1.3.0 doesn't deserialize the primitive type. We need to fix them ;)

**Comment by Romakita** [comment]
Hi @zoubarevm ,

Fix in v1.3.2. Now String, Number and Boolean will be deserialized correctly.

```typescript
@PathParams('id') id: number => Number 
// if id = 1 => 1
// if id = "1" => 1 
// if id = "zefze1" => throw BadRequest 

@PathParams('id') id: string => String
// if id = 1 => "1"
// if id = "1" => "1" 

@PathParams('id') id: boolean => Boolean  
// if id = 1 => true
// if id = "1" => true
// if id = "true" => true
// if id = "false" => false
// etc...
```
> Note: Number deserialization can throw a BadRequest when casting failed.

See you :)

**Comment by zoubarevm** [comment]
@Romakita awesome! Just tried it and it looks awesome.

Thanks!
